tests/test_hinfo.py

```diff
@@ -0,0 +1,81 @@
+import os
+import shutil
+import sqlite3
+import tempfile
+from pathlib import Path
+
+import pytest
+import yaml
+from lsst.consdb import hinfo, utils
+from lsst.resources import ResourcePath
+from sqlalchemy import MetaData, Table, select
+
+
+@pytest.fixture
+def tmpdir(scope="module"):
+    os.environ["INSTRUMENT"] = "LATISS"
+    tmpdir = Path(tempfile.mkdtemp())
+    try:
+        yield tmpdir
+    finally:
+        shutil.rmtree(tmpdir)
+
+
+@pytest.fixture
+def engine(tmpdir, scope="module"):
+    db_path = tmpdir / "test.db"
+    instruments = ["latiss"]
+    with sqlite3.connect(db_path) as conn:
+        conn.execute("CREATE TABLE schemas (name text, path text)")
+        for instrument in instruments:
+            schema = f"cdb_{instrument}"
+            sql = Path(__file__).parent / f"{schema}.sql"
+            if not sql.exists():
+                continue
+            schema_path = tmpdir / f"{schema}.db"
+            conn.execute(f"INSERT INTO schemas VALUES ('{schema}', '{schema_path}')")
+            with sqlite3.connect(schema_path) as schema_conn:
+                schema_conn.executescript(sql.read_text())
+            schema_conn.close()
+
+    os.environ["POSTGRES_URL"] = f"sqlite:///{db_path}"
+    hinfo.engine = utils.setup_postgres()
```

Comment:
Hm. Maybe we should rename this method, since it's obviously not setting up postgres...